ebusd-2.1.x/en/vaillant/08.hmu.csv

```diff
@@ -1,52 +1,156 @@
-!include,errors.inc,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,
+# type (r[1-9];w;u),circuit,name,[comment],[QQ],ZZ,PBSB,[ID],field1,part (m/s),datatypes/templates,divider/values,unit,comment,field2,part (m/s),datatypes/templates,divider/values,unit,comment,field3,part (m/s),datatypes/templates,divider/values,unit,comment,field4,part (m/s),datatypes/templates,divider/values,unit,comment
+*r,,,,,,"B511",,,,,,,,,,,,,,,,,,,,,,,,,
+r,,State,,,,,07,energy,,UCH,,,,,,energy,,kWh,,onoff,,UCH,0=off;1=on,,,state,,UCH,0x01=ready;0x0b=error;0x09=heating;0x11=cooling;0x81=heating_water,,
+# Heatpump statistics,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,
+*r,,,,,,"B51A","05E534",,,IGN:3,,,,,,,,,,,,,,,,,,,,,
+r,,CompressorHours,,,,,00,,,UIN,,hours,,,,,,,,,,,,,,,,,,,
+r,,CompressorStarts,,,,,01,,,UIN,,,,,,,,,,,,,,,,,,,,,
+r,,BuildingPumpHours,,,,,02,,,UIN,,hours,,,,,,,,,,,,,,,,,,,
+r,,BuildingPumpStarts,,,,,03,,,UIN,,,,,,,,,,,,,,,,,,,,,
+r,,4PortValveHours,,,,,06,,,UIN,,hours,,,,,,,,,,,,,,,,,,,
+r,,4PortValveSwitches,,,,,07,,,UIN,,,,,,,,,,,,,,,,,,,,,
+r,,EEVSteps,,,,,0D,,,UIN,,,,,,,,,,,,,,,,,,,,,
+r,,NoOfSwitchingOps,,,,,30,,,UIN,,,,,,,,,,,,,,,,,,,,,
+r,,Fan1OperHours,,,,,51,,,UIN,,hours,,,,,,,,,,,,,,,,,,,
+r,,Fan1Starts,,,,,52,,,UIN,,,,,,,,,,,,,,,,,,,,,
+r,,Fan2OperHours,,,,,5E,,,UIN,,hours,,,,,,,,,,,,,,,,,,,
+r,,Fan2Starts,,,,,5F,,,UIN,,,,,,,,,,,,,,,,,,,,,
+*r,,,,,,"B51A","05FF32",,,IGN:3,,,,,,,,,,,,,,,,,,,,,
+r,,YieldHcDay,,,,,00,,,energy,10,kWh,,,,,,,,,,,,,,,,,,,
+r,,YieldCoolingDay,,,,,01,,,energy,10,kWh,,,,,,,,,,,,,,,,,,,
+r,,YieldHwcDay,,,,,02,,,energy,10,kWh,,,,,,,,,,,,,,,,,,,
+r,,ch_3,,,,,03,,,UIN,,,,,,,,,,,,,,,,,,,,,
+r,,ch_4,,,,,04,,,UIN,,,,,,,,,,,,,,,,,,,,,
+r,,ch_5,,,,,05,,,UIN,,,,,,,,,,,,,,,,,,,,,
+r,,ch_6,,,,,06,,,UIN,,,,,,,,,,,,,,,,,,,,,
+r,,ch_7,,,,,07,,,UIN,,,,,,,,,,,,,,,,,,,,,
+r,,YieldHcMonth,,,,,0e,,,energy,10,kWh,,,,,,,,,,,,,,,,,,,
+r,,CopHcMonth,,,,,0f,,,UIN,10,,,,,,,,,,,,,,,,,,,,
+r,,YieldHc,,,,,10,,,energy,,kWh,,,,,,,,,,,,,,,,,,,
+r,,CopHc,,,,,11,,,UIN,10,,,,,,,,,,,,,,,,,,,,
+r,,YieldHwcMonth,,,,,12,,,energy,10,kWh,,,,,,,,,,,,,,,,,,,
+r,,CopHwcMonth,,,,,13,,,UIN,10,,,,,,,,,,,,,,,,,,,,
+r,,YieldHwc,,,,,16,,,energy,,kWh,,,,,,,,,,,,,,,,,,,
+r,,CopHwc,,,,,17,,,UIN,10,,,,,,,,,,,,,,,,,,,,
+r,,ch_18,,,,,18,,,UIN,,,,,,,,,,,,,,,,,,,,,
+r,,TargetTempHc,,,,,1C,,,D2C,,°C,Target temperature Hc (°C),,,,,,,,,,,,,,,,,,
+# Uncomment the next line if you want to be able to change TargetTempHc (from Home Assistant for example)
+#w,hmu,TargetTempHc,,,08,B51A,06B5321C,,,D2C,,°C,Target temperature Hc(°C),,,,,,,,,,,,,,,,,,
+r,,ch_1e,,,,,1e,,,UIN,,,,,,,,,,,,,,,,,,,,,
+r,,SupplyTempWeighted,,,,,1F,,,D2C,10,°C,,,,,,,,,,,,,,,,,,,
+r,,FlowTemp,,,,,20,,,D2C,,°C,,,,,,,,,,,,,,,,,,,
+r,,Integral,,,,,21,,,SIN,,,,,,,,,,,,,,,,,,,,,
+r,,SourceTempInput,,,,,22,,,D2C,,°C,,,,,,,,,,,,,,,,,,,
+r,,CurrentYieldPower,,,,,23,,,D1B,10,kW,,,,,,,,,,,,,,,,,,,
+r,,CurrentConsumedPower,,,,,24,,,D1B,10,kW,,,,,,,,,,,,,,,,,,,
+r,,CompressorUtilization,,,,,25,,,UIN,10,%,,,,,,,,,,,,,,,,,,,

```

Comment:
For HMU00;0607;5103, UIN,16 is dead-on for me. UIN,10 gives me readings up to around 110%.

Reply:
D2C also works fine for me.

Reply:
> D2C also works fine for me.

@pulquero : what type of system do you have? I also have a split, so could change the register to D2C as proposed, but don't want to create issues for owners of a monoblock.

Reply:
thanks for confirming. i've updated the registor in my repository